common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -295,15 +312,25 @@ object GettyXmpParser extends ImageProcessor with GettyProcessor {
 
     val isExcludedByCredit = image.metadata.credit.exists(isExcluded(_, excludedCredit))
     val isExcludedBySource = image.metadata.source.exists(isExcluded(_, excludedSource))
-    val hasGettyMetadata = image.fileMetadata.getty.nonEmpty
+    val hasGettyMetadata = image.fileMetadata.getty.nonEmpty && getSuppliersReference(image).nonEmpty
```

Comment:
```suggestion
    val hasGettyMetadata = image.fileMetadata.getty.nonEmpty
```

Reply:
Yep! Thanks. Let’s apply it separately when we will be making whole Getty business more sane.